index.html

```diff
@@ -5,13 +5,14 @@
   <title>Mobile Version</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel='stylesheet' type='text/css' href='styles.css'>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;700&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
```

Comment:
- [ ] This should be a section tag like the requirement says 
![fff](https://user-images.githubusercontent.com/61562626/129428121-3b2bfdf5-4ad2-41c2-bf43-39bf8c17b22e.jpeg)